dwave/system/samplers/leap_hybrid_sampler.py

```diff
@@ -945,10 +945,16 @@ def sample(self, model: dwave.optimization.Model,
 
         Returns:
             :class:`~concurrent.futures.Future` [SampleResult]:
-                Named tuple containing nonlinear model and timing info, in a Future.
+                Named tuple containing nonlinear model and general result info
+                (such as timing information and problem data id), in a Future.
+
+        .. versionchanged:: 1.31.0
+            The return type includes timing information as part of the ``info``
+            field dictionary, which now replaces the old ``timing`` field.
```

Comment:
```suggestion
            The return value includes timing information as part of the ``info``
            field dictionary, which now replaces the previous ``timing`` field.
```

I think the return type remains a named tuple but the value changes

Reply:
No, type changed as well, as it's defined with fields of the named tuple.